EIPS/eip-4521.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 4521
+title: 721/20-compatible transfer
+description: Recommends a simple extension to make NFTs compatible with apps and contracts that handle fungibles.
+author: Ross Campbell (@z0r0z)
+discussions-to: https://github.com/ethereum/EIPs/issues/4521
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-12-13
+requires 20, 721
```

Comment:
```suggestion
requires: 721
```
I'd argue that this doesn't require ERC-20 since you don't need an ERC-20 interface for this, even if it's implementing a function based on ERC-20